config/config_params.go

```diff
@@ -33,11 +33,19 @@ import (
 )
 
 var (
-	IfaceListRegexp  = regexp.MustCompile(`^[a-zA-Z0-9_-]{1,15}(,[a-zA-Z0-9_-]{1,15})*$`)
-	AuthorityRegexp  = regexp.MustCompile(`^[^:/]+:\d+$`)
-	HostnameRegexp   = regexp.MustCompile(`^[a-zA-Z0-9_.-]+$`)
-	StringRegexp     = regexp.MustCompile(`^.*$`)
-	IfaceParamRegexp = regexp.MustCompile(`^[a-zA-Z0-9:._+-]{1,15}$`)
+	// RegexpIfaceListRegexp differs from IfaceListRegexp in that it allows both
+	// normal interface values but also regex pattern values (which are marked
+	// by '/' at the start and end) in the comma delimited list
+	RegexpIfaceListRegexp = regexp.MustCompile(`^([a-zA-Z0-9_-]{1,15}|(\/[^,\s]+\/))(,([a-zA-Z0-9_-]{1,15}|(\/[^,\s]+\/)))*$`)
+	// RegexpIfaceElemRegexp matches an individual element in the overall list;
+	// assumes the value is marked by '/' at the start and end and cannot have commas
+	// or spaces
+	RegexpIfaceElemRegexp = regexp.MustCompile(`^\/[^,\s]+\/$`)
```

Comment:
I think this can be just `^\/[^\s]+\/$`.  We don't need to prohibit commas, because the raw value has already been split at commas before applying this regexp.

Reply:
Fixed.